doc/genstdlib.jl

```diff
@@ -3,7 +3,7 @@ module GenStdLib
 import Base.Docs: Binding, DocStr
 
 # Constants.
-const DOCSTRING_DIRECTIVE = r"^(.. (function|type|variable):: ).*"
+const DOCSTRING_DIRECTIVE = r"^(.. (function|type|variable|data):: ).*"
```

Comment:
@MichaelHatherly I originally introduced the [`variable` directive](https://github.com/JuliaLang/JuliaDoc/blob/master/juliadoc/julia.py#L21) for the purpose of keeping docstrings in sync with the manual. We can definitely have the `data` directive sync with docstrings but then we should probably remove the `variable` directive.